feg/gateway/services/session_proxy/credit_control/gx/model_conversion.go

```diff
@@ -73,6 +81,8 @@ func (rd *RuleDefinition) ToProto() *protos.PolicyRule {
 		FlowList:      rd.GetFlowList(),
 		Qos:           rd.Qos.ToProto(),
 		TrackingType:  rd.GetTrackingType(),
+		Offline:	   offline,
+		Online: 	   online,
```

Comment:
Offline: rd.Offline != 0, // strictly speaking, it should be rd.Offline == 1
Online: rd.Online == 1,

Reply:
sure i can reverse it 

I was thinking to enable it for any value that were not 0. But maybe it is more correct to just enable it for explicit values = 1